src/components/Dashboard/FilingHistoryList/bodies/FutureEffective.vue

```diff
@@ -27,52 +31,73 @@
 import Vue from 'vue'
 import { Component, Prop } from 'vue-property-decorator'
 import { Getter } from 'vuex-class'
-import { DateMixin } from '@/mixins'
 import { ContactInfo } from '@/components/common'
-import { HistoryItemIF } from '@/interfaces'
+import { ApiFilingIF } from '@/interfaces'
+import { DateUtilities, EnumUtilities } from '@/services'
 
 @Component({
-  components: { ContactInfo },
-  mixins: [DateMixin]
+  components: { ContactInfo }
 })
 export default class FutureEffective extends Vue {
   @Getter getLegalName!: string
 
   /** The subject filing. */
-  @Prop({ required: true }) readonly filing!: HistoryItemIF
+  @Prop({ required: true }) readonly filing!: ApiFilingIF
 
   /** Data for the subject filing. */
   get _ (): any {
-    if (this.filing.isFutureEffectiveIa) {
+    if (EnumUtilities.isTypeIncorporationApplication(this.filing)) {
       return {
         subtitle: 'Future Effective Incorporation Date',
         filingLabel: 'incorporation',
         filingTitle: 'Incorporation Application'
       }
     }
```

Comment:
Hey, why we are we using `get _()` to determine the subtitle, filingLabel and filingTitle, when we've used `<component :is="" />' to separate the filings by filing type from the dashboard.  Isn't the filing type known at this point?

Reply:
Yes, the filing type is know in this partial. This is actually previous code that I didn't change. Believe it or not, I tried to not change every single thing I touched. It was really hard. I was very tempted.

In order to implement what you suggest, we could pass props (or use slots). However, I've reached my limit for things I want to change in this ticket (other than any bugs I created -- fingers crossed!).

Reply:
> However, I've reached my limit for things I want to change in this ticket...

Got it.  PR comments > /dev/null  :-)

Reply:
Haha, no, keep the comments coming, but don't add extra work :)

(I've been buried under several linked/blocked features/PRs for 3 sprints now. I'm almost at the surface and desperately need to breathe a little ;) )

Reply:
Yes, I totally get it.  I'll create refactoring tickets for issues that can't be resolved in the PR.